test/e2e/main_test.go

```diff
@@ -115,10 +117,10 @@ func testMainWrapper(m *testing.M) int {
 	// the default images in the metering CSV. In the case where
 	// some version of the `make e2e-dev` has been provided, skip
 	// the deletion the local registry resources and CatalogSource CR.
-	if meteringOperatorImageRepo != "" {
+	if meteringOperatorImageRepo != "" && meteringOperatorImageTag != "" {
```

Comment:
We have this kind of validation in place lower in the testing suite to ensure that a context's deployment doesn't use an empty repository or tag variable, but we need to ensure that we're not overriding any default images unless the repository and tag variables have been provided.